bootstrap/kubeadm/api/.import-restrictions

```diff
@@ -0,0 +1,5 @@
+rules:
```

Comment:
Let's also add an additional restrictions file into the v1beta package (the same as api/v1beta1/.import-restrictions)

same for the controlplane v1beta1 package